File: app/src/main/java/com/nexaflow/saayalite/domain/model/PoliceStation.kt
package com.nexaflow.saayalite.domain.model

import kotlinx.serialization.Serializable

@Serializable
data class PoliceStation(
    val id: String,
    val name: String,
    val category: String,
    val locality: String,
    val areaCovered: String,
    val latitude: Double,
    val longitude: Double,
    val coordPrecision: String,
    val phone: String,
    val address: String,
)

File: src/domain/model/policeStation.ts
/** A police or support point parsed from the bundled station asset. */
export interface PoliceStation {
  readonly id: string;
  readonly name: string;
  readonly category: string;
  readonly locality: string;
  readonly areaCovered: string;
  readonly latitude: number;
  readonly longitude: number;
  readonly coordPrecision: string;
  readonly phone: string;
  readonly address: string;
}
